rllib/policy/view_requirement.py

```diff
@@ -0,0 +1,55 @@
+from gym.spaces import Box, Space
+from typing import Optional
+
+from ray.rllib.utils.framework import try_import_torch
+
+torch, _ = try_import_torch()
+
+
+class ViewRequirement:
+    """Single view requirement (for one column in a ModelV2 input_dict).
+
+    Note: This is an experimental class.
+
+    ModelV2 returns a Dict[str, ViewRequirement] upon calling
+    `ModelV2.get_view_requirements()`, where the str key represents the column
+    name (C) under which the view is available in the `input_dict` and
+    ViewRequirement specifies the actual underlying column names (in the
+    original data buffer), timesteps, and other options to build the view
+    for N.
+
+    Examples:
+        >>> # The default ViewRequirement for a Model is:
+        >>> req = [ModelV2].get_view_requirements(is_training=False)
+        >>> print(req)
+        {"obs": ViewRequirement(timesteps=0)}
```

Comment:
Do we still have timesteps? Also, it makes sense to have timesteps=1 mean one timestep right?

Reply:
example was fixed